### PR TITLE
Add tidy github actions (from usethis)

### PR DESCRIPTION
**Pull request by thebioengineer**
Adds github actions to the repo based on usethis::use_tidy_github_actions()
- Run r cmd check on PR and merge to main for multiple OS's and R versions
- In PR's when "/document" or "/style" is commented, will document and stylize code and commit it
- run test coverage on push/PR to main
- build a pkgdown site & deploy on push to main or releases

Question: Do we plan to use gitflow? if so, we may want to add dev to these too
@statasaurus

### CODE REVIEW

**Comment by statasaurus** [comment]
> Adds github actions to the repo based on usethis::use_tidy_github_actions()
> 
> * Run r cmd check on PR and merge to main for multiple OS's and R versions
> * In PR's when "/document" or "/style" is commented, will document and stylize code and commit it
> * run test coverage on push/PR to main
> * build a pkgdown site & deploy on push to main or releases
> 
> Question: Do we plan to use gitflow? if so, we may want to add dev to these too @statasaurus

I don't plan on using gitflown. I want to try to keep pretty close to main/dev and not have a bunch of branches hanging out there for ages

File: .github/workflows/pr-commands.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
```
Comment (statasaurus) [inline .github/workflows/pr-commands.yaml:1]: This is a note for myself, but this action rebuilds the documentation and styles when submitting a pull request

**Review by statasaurus — APPROVED** [review]
Thanks for adding the actions

**Comment by thebioengineer** [comment]
Did you intend to close or merge this?

**Pull request merged**